src/kili/authentication.py

```diff
@@ -112,12 +112,14 @@ def check_expiry_of_key_is_close(self, api_key):
         """
         duration_days = 365
         warn_days = 30
-        key_object = APIKeyQuery(self.client)(
-            fields=["createdAt"],
-            where=APIKeyWhere(api_key=api_key),
-            options=QueryOptions(disable_tqdm=True),
+        api_keys = list(
+            APIKeyQuery(self.client)(
+                fields=["createdAt"],
+                where=APIKeyWhere(api_key=api_key),
+                options=QueryOptions(disable_tqdm=True),
+            )
         )
-        key_creation = datetime.strptime(list(key_object)[0]["createdAt"], "%Y-%m-%dT%H:%M:%S.%fZ")
+        key_creation = datetime.strptime(api_keys[0]["createdAt"], "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
alternatively, `next(api_keys)` when `api_keys` is a generator also works.

Reply:
comment taken